src/middlewared/middlewared/plugins/netdata.py

```diff
@@ -37,6 +37,27 @@ def __init__(self, *args, **kwargs):
             data['alarms'][alarm].pop('write_path', None)
         return data
 
+    @accepts()
+    async def ips(self):
+        """
+        Returns a list of user configured addresses with which netdata can be accessed.
+        """
+        ips = []
+        for ip in (await self.config())['bind']:
+            if ip == '0.0.0.0':
+                ips.extend([
+                    f'http://{d["address"]}/netdata/'
```

Comment:
Why not https?

Reply:
`http+https` is enabled by default in the system. If user wants to enforce `https`, he/she can just set the relevant settings in system.general and all requests would be proxied to `https`.

Reply:
I mean we always have https enabled and we are trying to enforce to use it by default, why http is the default value here?

Reply:
Copying `system.general` behavior where the default is to use `http` unless forced by the user. Please advise if you would like me to change this to `https`.